docs/changelog/0.4.0.md

```diff
@@ -0,0 +1,523 @@
+
+### Commix GC
+This release also adds a new Garbage Collector - Commix, a parallel mark, and concurrent sweep GC, based on the well known Immix GC.
+It reduces GC pause times by utilizing additional processor cores during mark and sweep phases. It was added soon after
+the 0.4.0-M2 release, but was only available via an unofficial snapshot.
+
+Remember, it **does not** mean Scala Native is starting to officially support multi-threading!
+Commix GC was written in C and uses `pthread` to work. In case if your application needs concurrency support you may try experimental library [scala-native-loop](https://github.com/scala-native/scala-native-loop)
+
+
+## Java standard library changes
+### Removal of java.time / java.text
+This release comes with removal of `java.time` and `java.text` packages for Scala Native core. This decision was taken
+in order to allow for proper implementation of those modules in third-party library. It's specially important for classes
+like `Locale` that are hard to maintain in Scala Native core repo.
+
+Removal of mentioned modules does not create regression in other definitions as long as they're not using directly or transitively one of removed classes.
+```scala
+val str: String = "Hello Native"
+str.toLowerCase()              // works as before
+str.toLowerCase(Locale.French) // fails while linking if not provided implementation
+```
+  
+
+### Unicode specification
+We have updated our `java.lang.String` and `java.lang.Character` to be compatible with Unicode 13.0.0 specification,
+allowing us to replace relatively old standard 6.3.0 used in JDK 8
+Additionally we've implemented missing handling of `java.lang.String.to{Lower, Upper}Case` methods for 
+special and language specific casing rules. Further one requires linking implementation for `Locale`. 
+
+## Summary
+<table>
+<tbody>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">196</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">191</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">15</td>
+  </tr>
+  <tr>
+</tbody>
+</table>
+
+[Full Changelog](https://github.com/scala-native/scala-native/compare/v0.4.0-M2...v0.4.0)
+
+Scala Native is an optimizing ahead-of-time compiler and lightweight managed runtime designed specifically for Scala. 
+It is developed at the [Scala Center](https://scala.epfl.ch/) and with the help from [VirtusLab](https://virtuslab.com)
+along with contributors from the community.
+
+## TL;DR
+
+Check out the documentation at [https://scala-native.readthedocs.io/](https://scala-native.readthedocs.io/en/latest)
+
+* **Not backward compatible with previous releases**,
+* Added Scala 2.12 and 2.13 support,
+* Added Native JUnit tests plugin,
+* Allowed for cross compilation using custom target triple
+* In sbt config `NativeConfig` replaces spare set of `NativeX` key settings,
+* Added new Garbage Collector - Concurrent Mark and Pararrell Sweep Garbage Collector, called Commix,
```

Comment:
Spelling of `Parallel`.

Reply:
Ahhh, again the same typo. 

Yes, I think they would be merged before release